src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -1,16 +1,57 @@
 package com.google.codeu.servlets;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+// Handles fetching and saving user data.
```

Comment:
Writiing class comments are usually in the form of a javadoc (https://google.github.io/styleguide/javaguide.html#s7-javadoc)

Something like 

```
/** A servlet that handles fetching and saving user data to display the About Me page. */
```